bin/i18n/hoc_sync_utils.rb

```diff
@@ -7,31 +7,6 @@
 require 'tempfile'
```

Comment:
Do we need this? Ideally, we would just refactor this module into the new name and location.

Reply:
We can use it i  `sync_out`, I'll check it when we create unit tests for sync-out

Reply:
That was my guess.